### PR TITLE
sql file + badges

### PR DESCRIPTION
**Pull request by tiffanynn**
*badge desc text file
* badges done
* sql file done

rebased already!

### CODE REVIEW

File: src/Pages/Shop/Shop.js [hunk 1]
```diff
@@ -1,12 +1,73 @@
 import React, { Component } from "react";
 import {
+<<<<<<< HEAD
```
Comment (realCalvin) [inline src/Pages/Shop/Shop.js:3]: merge conflict line 3-8

File: src/Pages/Shop/Shop.js [hunk 2]
```diff
@@ -1,12 +1,73 @@
...
+  Card, CardText, CardBody, CardTitle, CardSubtitle, CardImg,
+  Col,
+=======
   Card, CardText, CardBody, CardTitle, CardSubtitle, CardImg, CardFooter,
+>>>>>>> 57b49bb3dcd3f98a077d519be6ec92ec132e5943
```
Comment (realCalvin) [inline src/Pages/Shop/Shop.js:8]: merge conflict 8-15

File: src/Pages/Shop/Shop.js [hunk 3]
```diff
@@ -1,12 +1,73 @@
...
   Row,
   Button,
   Jumbotron
 } from "reactstrap";
 
 import ShopUser from './UserShop';
+<<<<<<< HEAD
```
Comment (realCalvin) [inline src/Pages/Shop/Shop.js:15]: conflict

File: src/Pages/Shop/Shop.js [hunk 4]
```diff
@@ -59,10 +120,29 @@ class Shop extends Component {
                     <Row>
                       <CardSubtitle ><b>Cost: </b>{shopbadge.badgeCost}</CardSubtitle>
                     </Row>
+>>>>>>> 57b49bb3dcd3f98a077d519be6ec92ec132e5943
```
Comment (realCalvin) [inline src/Pages/Shop/Shop.js:123]: conflict

File: src/Pages/Shop/Shop.js [hunk 5]
```diff
@@ -59,10 +120,29 @@ class Shop extends Component {
...
                     <CardText>
                       <Row style={{ height: '100px' }}>
                         {shopbadge.badgeDesc}
                       </Row>
+<<<<<<< HEAD
```
Comment (realCalvin) [inline src/Pages/Shop/Shop.js:128]: conflict

**Review by realCalvin — CHANGES_REQUESTED** [review]
conflicts in shop.js and shop.css. your branch is behind master. try rebasing again because jerry/habib recently pushed new stuff today

**Comment by jerrylee17** [comment]
@tiffanynn when you push to owo branch this pull request gets updated automatically

**Pull request closed without merging**